setup.cfg

```diff
@@ -25,7 +25,7 @@ classifiers =
 packages = find:
 python_requires = >=3.6
 install_requires =
-    google-cloud-logging < 2.0
+    google-cloud-logging < 4.0
```

Comment:
Due to the major library upgrade, I think we should bump the version to 3.0.0

Reply:
It is backwards compatible with 1.x and 2.x 🤷‍♂️

Reply:
But, I'm fine with that... I'm not familiar with how you all have been doing versioning. Was thinking major bumps were reserved for breaking changes

Reply:
[ac432f9](https://github.com/obsrvbl-oss/gcp-flowlogs-reader/pull/27/commits/ac432f9f4d9405b403c6a22d0e1cdebdb7b80822)